webapp/app/cognitoAuth.js

```diff
@@ -74,14 +103,9 @@ export const getJwtToken = () => new Promise((resolve, reject) => {
         reject(err)
       }
 
-      try {
-        const accessToken = session.getAccessToken()
-        const jwtToken = accessToken.jwtToken
-        resolve(jwtToken)
-      } catch (e) {
-        reject(e)
```

Comment:
any reason you removed the try catch here?

Reply:
No, it works in both cases but try catch seems more correct...